source/MapPanel.cpp

```diff
@@ -223,8 +225,15 @@ void MapPanel::Draw()
 {
 	glClear(GL_COLOR_BUFFER_BIT);
 
+	int buffer = FrameBuffer::CreateFrameBuffer();
```

Comment:
Create the framebuffer in the constructor (MapPanel class member that contains the framebuffer)

Reply:
That would be more efficient, yes